CMakeLists.txt

```diff
@@ -738,6 +738,7 @@ set(TIC80STUDIO_SRC
     ${TIC80LIB_DIR}/studio/editors/sfx.c
     ${TIC80LIB_DIR}/studio/editors/music.c
     ${TIC80LIB_DIR}/studio/studio.c
+    ${TIC80LIB_DIR}/studio/project.c
```

Comment:
`project.c` should only be built in Pro version, why did you add it here?

Reply:
I saw below you added this for the sake of `tic_project_ext()` function, maybe better move it to the `src/tools.c`

Reply:
I could... but curious, what is the harm in building it if it's never linked?  I don't see any real harm.

Reply:
Why should we include it in the project if its code is not used in the PRO build?
Another way, you can add it but pls wrap all the functions with 
```c 
#if defined(TIC80_PRO)
...
#endif
```

Reply:
>  if its code is not used in the PRO build?

_But it's code IS used in the build now_... so including it makes sense.  The functions that aren't referenced are never included always because the linker will just silently discard them.

You're way overthinking this one, but I went ahead and just moved the function to `tools`...